src/commands/General/mergepdf.ts

```diff
@@ -0,0 +1,42 @@
+import { ApplyOptions } from '@sapphire/decorators';
+import type {
+ Args, CommandOptions, Result, UserError,
+} from '@sapphire/framework';
+import type { Message } from 'discord.js';
+import PdfMerger from 'pdf-merger-js';
+import { mergePDF as config } from '@/config/commands/general';
+import MonkaCommand from '@/structures/MonkaCommand';
+import type { GuildMessage } from '@/types';
+
+@ApplyOptions<CommandOptions>(config.options)
+export default class PdfMergeCommand extends MonkaCommand {
+  public async run(message: GuildMessage, args: Args): Promise<void> {
+    // Argument dans l'ordre croissant
+    const result: Result<Message[], UserError> = await args.repeatResult('message');
```

Comment:
```suggestion
    const result = await args.repeatResult('message');
```
Le type est inférer automatiquement, pas besoin de le remettre :)